lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -0,0 +1,657 @@
+                else {
+                    expect(cohortsView.$('.message-actions .action-primary').length).toBe(0);
+                }
+                if (!hasDetails) {
+                    expect(cohortsView.$('.summary-items').length).toBe(0);
+                }
+            };
+
+            verifyNoMessage = function() {
+                expect(cohortsView.$('.message').length).toBe(0);
+            };
+
+            verifyDetailedMessage = function(expectedTitle, expectedMessageType, expectedDetails, expectedAction) {
+                var numDetails = cohortsView.$('.summary-items').children().length;
+                verifyMessage(expectedTitle, expectedMessageType, expectedAction, true);
+                expect(numDetails).toBe(expectedDetails.length);
+                cohortsView.$('.summary-item').each(function (index) {
+                    expect($(this).text().trim()).toBe(expectedDetails[index]);
+                });
+            };
+
+            verifyHeader = function(expectedCohortId, expectedTitle, expectedCount) {
+                var header = cohortsView.$('.cohort-management-group-header');
+                expect(cohortsView.$('.cohort-select').val()).toBe(expectedCohortId.toString());
+                expect(cohortsView.$('.cohort-select option:selected').text()).toBe(
+                    interpolate_text(
+                        '{title} ({count})', {title: expectedTitle, count: expectedCount}
+                    )
+                );
+                expect(header.find('.title-value').text()).toBe(expectedTitle);
+                expect(header.find('.group-count').text()).toBe(
+                    interpolate_ntext(
+                        '(contains {count} student)',
+                        '(contains {count} students)',
+                        expectedCount,
+                        {count: expectedCount}
+                    )
+                );
+            };
+
+            unknownUserMessage = function (name) {
+                return "Unknown user: " +  name;
+            };
+
+            beforeEach(function () {
+                setFixtures('<ul class="instructor-nav"><li class="nav-item"><<a href data-section="membership" class="active-section">Membership</a></li></ul><div></div>');
+                TemplateHelpers.installTemplate('templates/instructor/instructor_dashboard_2/cohorts');
+                TemplateHelpers.installTemplate('templates/instructor/instructor_dashboard_2/cohort-form');
+                TemplateHelpers.installTemplate('templates/instructor/instructor_dashboard_2/cohort-selector');
+                TemplateHelpers.installTemplate('templates/instructor/instructor_dashboard_2/cohort-editor');
+                TemplateHelpers.installTemplate('templates/instructor/instructor_dashboard_2/notification');
+                TemplateHelpers.installTemplate('templates/file-upload');
+            });
+
+            it("Show an error if no cohorts are defined", function() {
```

Comment:
Yes, we do. The intention is that it reads with the 'it', e.g. 'it shows an error if no cohorts are defined'. I've fixed it.